lib/improver/utilities/cube_manipulation_new.py

```diff
@@ -158,10 +97,179 @@ def strip_var_names(cubes):
     return cubes
```

Comment:
Are there likely to unconsidered implications for the threshold coordinate, which will only be identified as such by its var-name under our new metadata conventions, that result from this function in concatenating?

Reply:
Yes there are implications; no they are not unconsidered :)  I'm working on this in https://github.com/metoppv/improver/pull/838, so was planning to get that change through review and then add it to the "copy" in a separate PR.  However if my solution in the other PR looks OK to you (essentially exclude removal if `var_name == "threshold"`), I'm happy to include it here.